test/dotnet-vstest.Tests/VSTestTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using System;
+using System.IO;
+using FluentAssertions;
+using Microsoft.DotNet.TestFramework;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Cli.VSTest.Tests
+{
+    public class VSTestTests: TestBase
+    {
+        [Fact]
+        public void RunTestsFromGivenContainer()
```

Comment:
We generally use test names that express both the conditions the test is testing, and the expected result.

so, maybe change the name to something like `Running_tests_from_a_given_container_passes_and_has_expected_output` ?